packages/react-data-grid/src/ColumnUtils.js

```diff
@@ -43,5 +43,20 @@ module.exports = {
       value = column[property];
     }
     return value;
+  },
+
+  getWidthOfColumns: function getSize(columns, startIdx, endIdx) {
+    let totalWidth = 0;
+    if (Array.isArray(columns)) {
+      for (let idx = startIdx; idx <= endIdx; idx++) {
+        totalWidth += columns[idx].width;
```

Comment:
I don't think we can rely on columns always having an explicit `width` here.

Can you add some unit tests -- where all columns have widths, only some have widths, and none have widths?

Reply:
@jpdriver I've tested this locally (still need to make unit tests) and width always has a value. The library has a hard-coded minWidth of 80, so a column won't be smaller than that unless you specify one explicitly. Columns without explicit widths set act as if they were each `flex: 1`.

In the current implementation of `getRenderedColumnCount`, the column widths are already being accessed in the same way my new util function does. Each iteration, the column metrics for the given index are retrieved and the width is subtracted from `remainingWidth`.
```
getRenderedColumnCount(displayStart, width) {
    let remainingWidth = width && width > 0 ? width : this.props.columnMetrics.totalWidth;
    if (remainingWidth === 0) {
      remainingWidth = ReactDOM.findDOMNode(this).offsetWidth;
    }
    let columnIndex = displayStart;
    let columnCount = 0;
    while (remainingWidth > 0) {
      let column = ColumnUtils.getColumn(this.props.columnMetrics.columns, columnIndex);

      if (!column) {
        break;
      }

      columnCount++;
      columnIndex++;
      remainingWidth -= column.width;
    }
    return columnCount;
  }
```